packages/blockchain-wallet-v4-frontend/src/modals/RecoveryPhrase/ShowRecoveryWords/template.tsx

```diff
@@ -80,7 +80,7 @@ const WordsList = ({ handleBackArrow, handleNextButton, step, words }) => {
           if (index >= 6) {
             return (
               <WordBox>
-                <WordText>
+                <WordText data-e2e='backupWords'>
```

Comment:
these need to be unique otherwise how are they doing to target them... 
`data-e2e={``backupWord${index}``}`

Reply:
nevermind i see whats going on here. this is okay 👍